src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts

```diff
@@ -173,4 +177,11 @@ export class UbsUserProfilePageComponent implements OnInit {
   getControl(control: string) {
     return this.userForm.get(control);
   }
+
+  toggleAlternativeEmail() {
+    const control = new FormControl(this.userProfile?.alternateEmail, [Validators.pattern(this.regexpEmail)]);
```

Comment:
You should add   [Validators.required,...] , otherwise the user will be able to send an empty field